tests/metadata_views_tests.cdc

```diff
@@ -0,0 +1,90 @@
+import Test
+import "test_helpers.cdc"
+
+// access(all) let blockchain = Test.newEmulatorBlockchain()
+access(all) let sourceAccount = blockchain.createAccount()
+access(all) let accounts: {String: Test.TestAccount} = {}
+
+access(all) let exampleToken = "ExampleToken"
+
+/* Test Cases */
+
+access(all) fun testSetupAccountUsingFTView() {
+    let alice = blockchain.createAccount()
+    let bob = blockchain.createAccount()
+
+    setupExampleToken(alice)
+    let aliceBalance = getExampleTokenBalance(alice)
+    txExecutor("metadata/setup_account_from_vault_reference.cdc", [bob], [alice.address, /public/exampleTokenVault], nil, nil)
+    let bobBalance = getExampleTokenBalance(alice)
+
+    Test.assertEqual(0.0, aliceBalance)
+    Test.assertEqual(0.0, bobBalance)
+}
+
+access(all) fun testRetrieveVaultDisplayInfo() {
+    let alice = blockchain.createAccount()
+
+    setupExampleToken(alice)
+    let result = scriptExecutor("test/example_token_vault_display_strict_equal.cdc", [alice.address])! as! Bool
+
+    Test.assertEqual(true, result)
+}
+
+
+/* Transaction Helpers */
+
+access(all) fun setupExampleToken(_ acct: Test.TestAccount) {
+    txExecutor("setup_account.cdc", [acct], [], nil, nil)
+}
```

Comment:
I see this and other helpers in multiple test files. Would it make sense to put them in a single file so we don't have duplicate code?